matrix-json.c

```diff
@@ -231,3 +231,15 @@ GString *matrix_canonical_json(JsonObject *object)
 {
     return canonical_json_object(object, NULL);
 }
+
+/* Just dump the Json with the string prefix for debuggin */
+void matrix_debug_jsonobject(const char *reason, JsonObject *object)
+{
+    JsonNode *tmp_top = json_node_new(JSON_NODE_OBJECT);
+    json_node_set_object(tmp_top, object);
+    JsonGenerator *generator = json_generator_new();
+    json_generator_set_pretty(generator, TRUE);
+    json_generator_set_root(generator, tmp_top);
+    const char *json = json_generator_to_data(generator, NULL);
+    fprintf(stderr, "%s: %s\n", reason, json);
+}
```

Comment:
Should find that's fixed in the new version I just pushed, and the 'g's missing.